src/Symfony/Component/Intl/Data/Provider/CurrencyDataProvider.php

```diff
@@ -13,7 +13,6 @@
 
 use Symfony\Component\Intl\Data\Bundle\Reader\BundleEntryReaderInterface;
 use Symfony\Component\Intl\Exception\MissingResourceException;
-use Symfony\Component\Intl\Locale;
```

Comment:
Erm... are you sure about this?

Reply:
yep :) 

https://github.com/symfony/symfony/blob/70a941ec0c22c45c3bf2c47dec6bbd95882be283/src/Symfony/Component/Intl/ResourceBundle/CurrencyBundleInterface.php#L25-L26

updated as such below

it's a tiny improvement, but worth clarifying IMHO to rely on the root `\Locale` (either the real one, or the `Intl\Locale\Locale` stub.

`Intl\Locale` extends from `\Locale` and doesnt own the default, only the default fallback.

This internal, first citizent, `Intl\Locale` is super confusing, and is really a `LocaleFallback` util